File: C++/KelasBioskop.cpp
#ifndef KELASBIOSKOP_H
#define KELASBIOSKOP_H

#include <string>

using namespace std;

class KelasBioskop {
    private:
        string id;
        string namaKelas;
        int jumlahKursi;
        double harga;

    public:
        //constructor kosong
        KelasBioskop(){ }

        //constructor 
        KelasBioskop(string idInput, string nama, int kursi, double hrg){
            this->id = idInput;
            this->namaKelas = nama;
            this->jumlahKursi = kursi;
            this->harga = hrg;
        }

        //getter
        string getId() const { return id; }
        string getNamaKelas() const { return namaKelas; }
        int getJumlahKursi() const { return jumlahKursi; }
        double getHarga() const { return harga; }

        //setter (tanpa validasi)
        void setId(string idInput){ id = idInput; }
        void setNamaKelas(string nama){ namaKelas = nama; }
        void setJumlahKursi(int kursi){ jumlahKursi = kursi; }
        void setHarga(double hrg){ harga = hrg; }

        //destruktor
        ~KelasBioskop(){ }
};

#endif

File: Python/KelasBioskop.py
class KelasBioskop:
    #constructor
    def __init__(self, id_input: str = "", nama: str = "", kursi: int = 50, harga: float = 0.0):
        self._id = id_input #menyimpan id inputan
        self._nama_kelas = nama #menyimpan nama
        self._jumlah_kursi = kursi #menyimpan jumlah kursi yang ada
        self._harga = harga #menyimpan harga tiket

    # Getter (Mengambil Nilai)
    def get_id(self) -> str:
        return self._id

    def get_nama_kelas(self) -> str:
        return self._nama_kelas

    def get_jumlah_kursi(self) -> int:
        return self._jumlah_kursi

    def get_harga(self) -> float:
        return self._harga

    # Setter (Mengubah Sebuah Nilai)
    def set_id(self, id_input: str):
        self._id = id_input

    def set_nama_kelas(self, nama: str):
        self._nama_kelas = nama

    def set_jumlah_kursi(self, kursi: int):
        self._jumlah_kursi = kursi

    def set_harga(self, harga: float):
        self._harga = harga